### PR TITLE
Feature/ [Login/Register] [Header] [Contact/About Page]

### PR DESCRIPTION
**Pull request by Luke7787**
**Note: This pull request is very important! READ CAREFULLY**

**Summary:**

- **Updated** "Sign In" & "Sign Up" Modal to new UI design
- **Updated** Header w/ style and add "Home" button
- **Pressing** "MY INVENTORY" in header now takes us to "http://localhost:3000/inventory"
- **Created** "About" & "Contact" Page
- **Added** "Try Our Product" Picture to fill up place holder in our HomePage

**How to test:**
npm start

**Testing:**
- HomePage New Picture is positioned well - Tested ✓
- Sign In Modal work - Tested ✓
- Sign Up Modal work - Tested ✓
- Pressing "MY INVENTORY" in header - Tested ✓
- "About" page & "Contact" page works - Tested ✓

**VERY IMPORTANT NOTES**
- pressing login w/ empty Username and Password **[error]**
- Entering Username in Sign Up Modal **[error]**
- **A very weird bug** [ContactPage.tsx & AboutPage.tsx] **Read Bottom**

The styles for ContactPage & AboutPage doesn't load properly unless you randomly modify the file and restore it. 
For example at the top of both files I have "import React from 'react';" if i delete the t from the end ['reac'] and restore it back to ['react'] now the program will compile the new correct style. All you have to do is delete something and restore it back to its original code which is weird. I have already made sure check all imports. Update both files so class styles have different unique name. and even did some research and even ran this "npm install @babel/plugin-proposal-private-property-in-object --save-dev" all of which doesn't work still :(

### CODE REVIEW

**Comment by AnsonWooo** [comment]
Good job Luke! The redesign of the UI is looking good!

File: src/CreateAccount/CreateAccount.tsx [hunk 1]
```diff
@@ -212,69 +212,91 @@ const CreateAccount = () => {
...
 				<div className={styles.grid}>
 					<div className={styles.first}>
-						<p>First name</p>
 						<FormControl fullWidth>
 							<TextField
 								error={!!firstErr}
 								id="firstName-input"
 								helperText={firstErr ? firstErr : ''}
-								label="First name"
+								label={<span style={{ color: 'black', fontSize: '25px', fontFamily: 'Sniglet, cursive' }}>First Name</span>}
```
Comment (BaldoHado) [inline src/CreateAccount/CreateAccount.tsx:223]: Could you change this so we don't use inline styling?

File: src/Header/Header.module.scss [hunk 2]
```diff
@@ -1,15 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@700&display=swap');
```
Comment (BaldoHado) [inline src/Header/Header.module.scss:1]: In the future, we might want to keep track of all of our different fonts.

File: src/Header/Header.tsx [hunk 3]
```diff
@@ -1,61 +1,67 @@
...
-			</div>
-		</nav>
-	);
+  return (
+    <nav className={styles.header}>
+      <div className={styles.headerLeft}>
+      <div onClick={() => navigate('/inventory')} style={{ cursor: 'pointer' }}>
+          <h1>My Inventory</h1>
+        </div>
+        <img src={`${process.env.PUBLIC_URL}/images/newlogo.png`} alt="Logo" className={styles.logoImage} />
```
Comment (BaldoHado) [inline src/Header/Header.tsx:21]: Can you provide the env file associated with this variable?

**Comment by AnsonWooo** [comment]
Overall the Header looks good, I think that implementing a profile would be good for the header.

**Pull request merged**